src/libraries/System.Private.CoreLib/src/System/Environment.Linux.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
For other Unix OS like {Free,Net}BSD default support, can this be renamed to `Environment.Unix.cs` and conditionally include `Environment.OSX.cs` in project file for macOS?

Reply:
Maybe:

`Environment.OSVersion.Unix.cs`
`Environment.OSVersion.OSX.cs`

Reply:
I can add `OSVersion` to the file name. For OSX and `Unix vs. `Linux`, I was following the same pattern as FileStream:

https://github.com/dotnet/runtime/blob/16b7db27c98dce5b4d15894db13acd629b8fea62/src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems#L1759-L1760

Do you think that should be changed as well?

Reply:
> Do you think that should be changed as well?

It would be nice.